lib/ui/screens/register_screen.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:summaries_app/ui/styles/app_colors.dart';
+import 'package:summaries_app/ui/widgets/app_app_bar.dart';
+import 'package:mask_text_input_formatter/mask_text_input_formatter.dart';
+import 'package:summaries_app/ui/widgets/app_drawer.dart';
+
+class RegisterScreen extends StatefulWidget {
+  const RegisterScreen({Key? key}) : super(key: key);
+
+  @override
+  _RegisterScreenState createState() => _RegisterScreenState();
+}
+
+class _RegisterScreenState extends State<RegisterScreen> {
+  final _formKey = GlobalKey<FormState>();
+  final TextEditingController _nameController = TextEditingController();
+  final TextEditingController _phoneController = TextEditingController();
+  final TextEditingController _emailController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+
+  get onPressed => null;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Column(
+          children: [buildTitle(), buildForm()],
+        ),
+      ),
+    );
+  }
+
+  buildTitle() {
+    return Container(
+      width: 800,
+      height: 250,
```

Comment:
não usar valores fixos. Crie abaixo da linha 25 "final Size size = MediaQuery.of(context).size;" e utilize o "(size.width ou size.height) * . 'algum numero' , essa variavel size tem as dimensões da tela do celular que abrir o app então utilize ela no lugar dos valores fixos.

Reply:
exemplo: height: size.width * .2 vai usar como altura 20% da largura da tela do celular